src/Raven.Studio/typescript/commands/database/tasks/geAiModelsCommand.ts

```diff
@@ -0,0 +1,27 @@
+import commandBase = require("commands/commandBase");
+import yup = require("yup");
+import endpoints = require("endpoints");
+
+class geAiModelsCommand extends commandBase {
+    constructor(readonly dto: AiModelsRequestDto) {
+        super();
+    }
+
+    execute(): JQueryPromise<string[]> {
+        const url = endpoints.global.studioTasks.studioTasksAiModels;
+
+        return this.post<string[]>(url, JSON.stringify(this.dto), null).then(this.mapToModelsArray);
```

Comment:
Failure handling is missing
```suggestion
        return this.post<string[]>(url, JSON.stringify(this.dto), null)
        .done(this.mapToModelsArray)
        .fail((response: JQueryXHR) => this.reportError("ERRORTEXT", response.responseText, response.statusText));
```

Reply:
I intentionally skiepd this. We do not want to display an error every time the models EP fails.